spec/requests/users_controller_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    before(:example) { get '/users' }
+    it 'responds with code 200' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'renders users index view' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'shows correct content in the view' do
+      expect(response.body).to include('List of all usres')
```

Comment:
* I get errors when I run the tests, below is a screenshot of the said error, Kindly fix these errors  
![image](https://github.com/yemidada/blog-app/assets/52497246/be91425b-cb53-4e62-9846-697eb3ab0e0d)